app/controllers/admin/featured_policies_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Admin::FeaturedPoliciesController < Admin::BaseController
+  before_filter :load_organisation
+
+  def index
+    @featured_policies = @organisation.featured_policies.order(:ordering)
+    @policies = Policy.all.sort_by(&:title)
+  end
+
+  def create
+    @featured_policies = @organisation.featured_policies.build(policy_content_id: params[:policy_content_id])
+    if @featured_policies.save
+      redirect_to admin_organisation_featured_policies_path(@organisation), notice: 'Policies featured'
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    @feature = @organisation.featured_policies.find(params[:id])
```

Comment:
Don't need an instance variable, and we should make the name consistent with the `create` action.